dist/logic.js

```diff
@@ -0,0 +1,27 @@
+class ExpenseManager {
+    constructor() {
+        this.expenses = []
+        this.newExpense = {}
+    }
+
+    async addExpense(name,amount,group,date) {
+        const addObjExpenses = {
+            name: name,
+            amount: amount,
+            group: group,
+            date: date
+        }
+        this.newExpense = addObjExpenses
+       await $.post("/expense", addObjExpenses)
+    }
+
+    async getExpenses(d1, d2) {
+        const getAllExpenses = await $.get(`/expenses/?d1=${d1}&d2=${d2}`)
+        this.expenses = []
+        getAllExpenses.forEach(expense => {
+            this.expenses.push(expense)
+        });
```

Comment:
You can just assign `getAllExpenses` directly to `this.expenses`.
No need to do a forEach